staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -243,6 +247,27 @@ var (
 				Weight: ptr.To(int32(60)),
 			},
 		})
+	// event priority-level. This is a relatively low priority configuration, loss of events
+	// is a relatively low harm thing.
+	SuggestedPriorityLevelConfigurationEvent = newPriorityLevelConfiguration(
+		"event",
+		flowcontrol.PriorityLevelConfigurationSpec{
+			Type: flowcontrol.PriorityLevelEnablementLimited,
+			Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
+				NominalConcurrencyShares: ptr.To(int32(5)),
+				LendablePercent:          ptr.To(int32(0)),
+				BorrowingLimitPercent:    ptr.To(int32(100)),
```

Comment:
Do we still need borrowing limit with weighted borrowing feature?

Reply:
Yes, I think so. For example, this makes it possible to "jail" some requests to a priority level with a fixed upper limit.